tables/elb_access_log/elb_access_log.go

```diff
@@ -0,0 +1,214 @@
+package elb_access_log
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/turbot/tailpipe-plugin-sdk/schema"
+)
+
+type ElbAccessLogBatch struct {
+	Records []ElbAccessLog `json:"Records"`
+}
+
+type ElbAccessLog struct {
```

Comment:
Are we able to get which region the ELB is in?

Reply:
No from the Application Load Balancer Logs, we do not get the region